packages/flutter/lib/src/painting/image_stream.dart

```diff
@@ -266,6 +324,14 @@ abstract class ImageStreamCompleter extends Diagnosticable {
   /// then use this flag to avoid calling [RenderObject.markNeedsPaint] during
   /// a paint.
   ///
+  /// An [ImageChunkListener] can also optionally be added along with the
+  /// `listener`. If the stream produces images that are loaded in chunks
+  /// (e.g. over a file system or a network), then the specified listener will
+  /// be notified for every chunk of data that is loaded. The events that are
+  /// delivered to the listener are _not_ buffered (rather, they are fired and
+  /// forgotten), so it'd possible that the listener will have missed delivery
+  /// of some [ImageChunkEvent]s by the time they add themselves as a listener.
```

Comment:
```suggestion
  /// of some [ImageChunkEvent]s by the time it is added as a listener.
```
(no need to anthropomorphize)